game.cpp

```diff
@@ -2126,17 +2145,16 @@ void Game::CountItems (int ** citems)
 			{
 				citems[i][j] = CountItem (fac, j);
 			}
-			
-			i++;
 		}
+		i++;
```

Comment:
since items for npc facs get initialized to 0, leaving the npc rows in the data made it easier to avoid recomputing the factions current item totals in the rank calculator.  So now the first player faction will be the third row (assuming guard and monster npc factions) of the citems array rather than the first.  This lets me just use faction->num - 1 to reference that factions data in citems consistently.